pom.xml

```diff
@@ -66,7 +66,7 @@
     <junit.version>4.12</junit.version>
     <sslr.version>1.22</sslr.version>
     <assertj.version>3.17.2</assertj.version>
-    <analyzerCommons.version>1.10.3.509</analyzerCommons.version>
+    <analyzerCommons.version>1.12.0.632</analyzerCommons.version>
```

Comment:
this makes me think that on pl-sql... we didn't changed the version and we are still relying on 1.9. Does it mean that hotspots are not working?!

Reply:
I had the same reaction, and checked PL/SQL: good news, everything is working as expected, as we are providing the `SonarRuntime` to `RuleMetadataLoader`.